lockbox/programs/liquidity_lockbox/src/lib.rs

```diff
@@ -503,11 +483,11 @@ pub struct DepositPositionForLiquidity<'info> {
     payer = signer)]
   pub pda_lockbox_position: Box<Account<'info, LockboxPosition>>,
 
-  #[account(mut)]
+  #[account(mut, address = lockbox.bridged_token_mint)]
   pub bridged_token_mint: Box<Account<'info, Mint>>,
   #[account(mut,
     constraint = bridged_token_account.mint == lockbox.bridged_token_mint,
-    constraint = bridged_token_mint.key() == lockbox.bridged_token_mint,
+    constraint = bridged_token_account.mint == bridged_token_mint.key(),
```

Comment:
Switching context of verified accounts to correspond to actual declared accounts.